mobile/App.tsx

```diff
@@ -57,6 +63,10 @@ export default class App extends React.Component<{}, {}> {
     PushNotifications.configure({
 
       onRegister: ({ token }: TokenResponse) => {
+        this.setState({
+          deviceId: token
+        });
+
         // Need to wait for the store to be setup
         setTimeout(async () => {
```

Comment:
this is pretty hacky :hankey:  You use the store already, so why not subscribe to store and wait explicitly for `accounts`?

Reply:
store is not available in App.tsx sadly... spent a while trying to get it in there.